packages/gestalt/src/Masonry.tsx

```diff
@@ -608,11 +640,9 @@ export default class Masonry<T> extends ReactComponent<Props<T>, State<T>> {
       const itemsToMeasureCount = hasMultiColumnItems
         ? MULTI_COL_ITEMS_MEASURE_BATCH_SIZE + 1
         : minCols;
-      const itemsToMeasure = items
-        .filter((item) => item && !measurementStore.has(item))
-        .slice(0, itemsToMeasureCount);
+      const itemsToMeasure = items.slice(0, itemsToMeasureCount);
```

Comment:
Is this correct? That we remove the filter for items that have not been measured?

Reply:
fixed 👍